pkg/ingester/lifecycle_test.go

```diff
@@ -419,10 +422,8 @@ func TestV2IngesterTransfer(t *testing.T) {
 	limits, err := validation.NewOverrides(defaultLimitsTestConfig())
 	require.NoError(t, err)
 
-	dir1, err := ioutil.TempDir("", "tsdb")
-	require.NoError(t, err)
-	dir2, err := ioutil.TempDir("", "tsdb")
-	require.NoError(t, err)
+	dir1 := filepath.Join(os.TempDir(), fmt.Sprintf("TestV2IngesterTransfer-%v", rand.Int63()))
```

Comment:
Out of curiosity: why do you need this change? Did you get failing or flaky tests with the previous code?

Reply:
ugh that was when I was debugging and I forgot to revert that. Will revert